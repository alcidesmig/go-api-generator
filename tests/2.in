Models {
	Notebook {
		name: string
	}
	User {
		nome: teste
		idade: float
		notebook: Notebook
	}
}

Routes {
	Models.User {
	# comment #
		GetUsers {
			Method: GOT
			Path: /user 			
		}
		GetUserByID {
			Method: GET
			Path: /user/:id
		}
		PostUser {
			Method: POST
			Path: /user
		}
		PutUser {
			Method: PUT
			Path: /user/:id
		}
		DeleteUser {
			Method: DELETE
			Path: /user/:id
		}
	}
	Models.Notebook {
		GetNotebooks {
			Method: GET
			Path: /notebooks
		}
	}
}
