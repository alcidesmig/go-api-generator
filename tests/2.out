Line 16: syntax error near GOT
End of compilation
